flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/planner/delegation/hive/HiveParserTypeCheckProcFactory.java

```diff
@@ -1216,7 +1216,7 @@ protected ExprNodeDesc getXpathOrFuncExprNodeDesc(
                     // Rewrite CASE into NVL
                     desc =
                             ExprNodeGenericFuncDesc.newInstance(
-                                    new GenericUDFNvl(),
+                                    new GenericUDFCoalesce(),
```

Comment:
I think it would be nice to adapt the comment @ line 1216 as well to keep it consistent. The change itself LGTM.

Reply:
I'm afriad of changing it may bring other bugs as reported int [HIVE-24902](https://issues.apache.org/jira/browse/HIVE-24902).
Also, I still pefer not to change it as I commented in the jira [FLINK-29432](https://issues.apache.org/jira/browse/FLINK-29432)

Reply:
Ack !